### PR TITLE
Resolving Issue #2572

### PR DESCRIPTION
**Pull request by j0rt3g4**
Taking notes on issue #2572 and transformed the paragraph into two notes (a simple note and an important one).

### CODE REVIEW

**Comment by j0rt3g4** [comment]
Resolving isue #2572

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/direct-routing-plan.md [hunk 1]
```diff
@@ -84,10 +84,14 @@ Users of Direct Routing must have the following licenses assigned in Office 365:
 - Microsoft Teams 
 - Microsoft Audio Conferencing 
 
-The Audio Conferencing license is required for adding external participants to scheduled meetings, either by dialing out to them or by providing the dial-in number. 
- 
-  > [!NOTE]
-  > The E5 license includes both Phone System and Audio Conferencing.   
+> [!NOTE]
+> When you are escalating from a 1:1 call the audio conferencing license is required
```
Comment (nenonix) [inline Teams/direct-routing-plan.md]:
```suggestion
> -  To escalate from 1:1 to group call you need an Audio Conferencing license.
> -  The E5 license includes both Phone System and Audio Conferencing.
```

File: Teams/direct-routing-plan.md [hunk 2]
```diff
@@ -84,10 +84,14 @@ Users of Direct Routing must have the following licenses assigned in Office 365:
...
+
+> [!IMPORTANT]
+>  In the case that you would like to add external participants to scheduled meetings, either by dialing out to them or by providing the dial-in number, the audio conferencing license is *required*.
+
+> [!NOTE]
```
Comment (nenonix) [inline Teams/direct-routing-plan.md]:
```suggestion
```

File: Teams/direct-routing-plan.md [hunk 3]
```diff
@@ -84,10 +84,14 @@ Users of Direct Routing must have the following licenses assigned in Office 365:
...
+> The E5 license includes both Phone System and Audio Conferencing.
```
Comment (nenonix) [inline Teams/direct-routing-plan.md]:
```suggestion
```

File: Teams/direct-routing-plan.md [hunk 4]
```diff
@@ -84,10 +84,14 @@ Users of Direct Routing must have the following licenses assigned in Office 365:
 - Microsoft Teams 
 - Microsoft Audio Conferencing 
 
-The Audio Conferencing license is required for adding external participants to scheduled meetings, either by dialing out to them or by providing the dial-in number. 
- 
-  > [!NOTE]
-  > The E5 license includes both Phone System and Audio Conferencing.   
+> [!NOTE]
```
Comment (nenonix) [inline Teams/direct-routing-plan.md]:
```suggestion
```

File: Teams/direct-routing-plan.md (hunk 1 above)
Comment (nenonix) [inline Teams/direct-routing-plan.md]:
```suggestion
```

File: Teams/direct-routing-plan.md [hunk 5]
```diff
@@ -84,10 +84,14 @@ Users of Direct Routing must have the following licenses assigned in Office 365:
...
+> When you are escalating from a 1:1 call the audio conferencing license is required
+
+> [!IMPORTANT]
+>  In the case that you would like to add external participants to scheduled meetings, either by dialing out to them or by providing the dial-in number, the audio conferencing license is *required*.
```
Comment (nenonix) [inline Teams/direct-routing-plan.md]:
```suggestion
>  The Audio Conferencing license is *required* to: 
> - Escalate from a 1:1 call to a group call.
> - Add external participants to scheduled meetings, by either dialing out or providing the dial-in number. 
```

**Comment by LolaJacobsen** [comment]
@CarolynRowe - for your review.

**Comment by j0rt3g4** [comment]
@nenonix  @CarolynRowe  @LolaJacobsen I've added the corrections into this new PR  #2641

**Pull request merged**